securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc

```diff
@@ -79,6 +87,10 @@ $ ./bin/jboss-cli.sh --file=adapter-elytron-install-saml.cli
 $ cd $JBOSS_HOME/bin
 $ jboss-cli.sh -c --file=adapter-install-saml.cli
 ----
+
+NOTE: It is possible to use legacy non-elytron adapter on JBoss EAP 7.1 or newer as well, hence you can use `adapter-install-saml.cli`
+even on those versions. However we recommend to use newer Elytron adapter.
+
```

Comment:
same note as before. I wonder whether we should pull this note out into a separate file and then include it in all these locations or whether that is more work than remembering to update it in four places in the future. I would lean toward pulling it out and including it, so that we only update in one place in the future.